tuplex/python/src/PythonContext.cc

```diff
@@ -1105,6 +1105,34 @@ namespace tuplex {
         return pds;
     }
 
+    PythonDataSet PythonContext::orc(const std::string &pattern,
+                                     boost::python::object cols) {
+        assert(_context);
+
+        // reset signals
+        if(check_and_forward_signals(true))
+            return makeError("job aborted via signal");
+
+        PythonDataSet pds;
```

Comment:
not 100% sure, but I remember that at some point we had a bug if we didn't surround the internal _context->orc with a try catch. Can you look at the other functions and add that?